src/utils/market.util.ts

```diff
@@ -0,0 +1,140 @@
+import { MarketCoinsSortFieldEnum } from '../enums/market-coins-sort-field.enum';
+import { MarketCoin, MarketCoinRaw } from '../store/market/market.interfaces';
+import { Colors } from '../styles/colors';
+import { coingeckoApi, templeWalletApi } from './../api.service';
+
+export const fetchMarketTopCoins = () =>
+  coingeckoApi
+    .get<Array<MarketCoinRaw>>(
+      'coins/markets?vs_currency=usd&category=tezos-ecosystem&order=market_cap_desc&per_page=100&page=1&sparkline=false&price_change_percentage=24h%2C7d'
```

Comment:
it's very hard to read such params string.  
pass all these params as second argument of `.get('url', { params })`